dotnet/README.MD

```diff
@@ -0,0 +1,179 @@
+ Cloud Bigtable and to create VM instances.
+ 
+ [enable-billing]: https://console.cloud.google.com/project/_/settings
+ 
+ ### Enable the Cloud Bigtable APIs.
+ 
+ Make sure to [enable the Bigtable APIs][enable-bigtable-api].
+ 
+ [enable-bigtable-api]: https://console.cloud.google.com/apis/library?q=bigtable
+ 
+ ### Install the Google Cloud SDK.
+ 
+ If you haven't already installed the Google Cloud SDK, [install the Google
+ Cloud SDK][cloud-sdk] now. The SDK contains tools and libraries that enable you
+ to create and manage resources on Google Cloud Platform.
+ 
+ [cloud-sdk]: https://cloud.google.com/sdk/
+ 
+ ### Setting Google Application Default Credentials
+ 
+ Set your [Google Application Default
+ Credentials][application-default-credentials] by [initializing the Google Cloud
+ SDK][cloud-sdk-init] with the command:
+ 
+     gcloud init
+ 
+ Generate a credentials file by running the [application-default login](https://cloud.google.com/sdk/gcloud/reference/auth/application-default/login) command:
+ 
+     gcloud auth application-default login
+ 
+ [cloud-sdk-init]: https://cloud.google.com/sdk/docs/initializing
+ [application-default-credentials]: https://developers.google.com/identity/protocols/application-default-credentials
+ 
+ 
+ ## Provisioning an instance
+ 
+ Follow the instructions in the [user
+ documentation](https://cloud.google.com/bigtable/docs/creating-instance) to
+ create a Google Cloud Platform project and Cloud Bigtable instance if necessary.
+ You'll need to reference your project id and instance id to run the
+ application.
+ 
+ 
+ ## Running the application
+ 
+ In appsettings.json replace `[PROJECT_ID]` with google cloud project name and `[BIGTABLEINSTANCE]` with what you called it when provisioning the instance.
+ 
+    "projectId": `[PROJECT_ID]`,
+    "instanceId": `[BIGTABLEINSTANCE]`,
+ 
+ Run the sample using `dotnet run` CLI command
+    
+    dotnet run -c Release
+ 
+ You will see output resembling the following, interspersed with informational logging
+ from the underlying libraries:
+ 
+    Creating new table: Hello-Bigtable
+    Table Hello-Bigtable created succsessfully
+
+    Writing some greetings to the table Hello-Bigtable
+    Reading the first row
+            Row key: greeting0, Value: Hello World!
+    Reading all rows using streaming
+            Row key: greeting0, Value: Hello World!
+            Row key: greeting1, Value: Hellow Bigtable!
+            Row key: greeting2, Value: Hellow C#!
+    Deleting table: Hello-Bigtable
+    Table Hello-Bigtable deleted succsessfully
+    Press any key to exit
+ 
+ 
+ ## Cleaning up
+ 
+ To avoid incurring extra charges to your Google Cloud Platform account, remove
+ the resources created for this sample.
+ 
+ 1.  Go to the Clusters page in the [Cloud
+     Console](https://console.cloud.google.com).
+ 
+     [Go to the Clusters page](https://console.cloud.google.com/project/_/bigtable/clusters)
```

Comment:
This should be s/instances/clusters/ and should be included in the link above — rather than having 2 separate links, one to GCP overall and one to Bigtable, just show the direct link to Bigtable so that there's no confusion.